src/pages/Governance/Stake/components/EditOperator.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect} from "react";
+import {
+  Grid,
+  Button,
+  FormControl,
+  Tooltip,
+  Stack,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+import TransactionResponseSnackbar from "../../components/snackbar/TransactionResponseSnackbar";
+import LoadingModal from "../../components/LoadingModal";
+import useAddressInput from "../../../../api/hooks/useAddressInput";
+import useSubmitChangeOperatorStake from "../../hooks/useSubmitChangeOperatorStake";
+import HashButton from "../../../../components/HashButton";
+
+type EditOperatorProps = {
+  isWalletConnected: boolean;
+  operator_address: string;
```

Comment:
```suggestion
  operatorAddress: string;
```
Also I've been using addr for short of address in case you want to make them consistent.